<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filePathLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="chooseFileButton">
        <property name="text">
         <string>Choose file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="fileContentTextEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>PartNumber:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="partNumberComboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="deleteItemButton">
        <property name="text">
         <string>Delete item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addItemButton">
        <property name="text">
         <string>Add item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveFileButton">
        <property name="text">
         <string>Save file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
